litmus-portal/frontend/src/views/Community/InfoFilled/index.tsx

```diff
@@ -1,16 +1,18 @@
-import { useTheme } from '@material-ui/core/styles';
+/* eslint-disable no-unused-expressions */
```

Comment:
can we remove this if possible?

Reply:
It coming because of the arrow function.